src/AppBundle/EventListener/RedirectToPreferredLocaleSubscriber.php

```diff
@@ -23,7 +25,7 @@
  *
  * @author Oleg Voronkovich <oleg-voronkovich@yandex.ru>
  */
-class RedirectToPreferredLocaleListener
+class RedirectToPreferredLocaleSubscriber implements EventSubscriberInterface
```

Comment:
Just asking: is this change needed because of Symfony ... or is it because you configured this: `resource: '../../src/AppBundle/{EventListener/*Subscriber.php,...}'`

In other words ... if the previous config was: `resource: '../../src/AppBundle/{EventListener,...}'` would this change be required?